### DISCUSSION TITLE
abstraction with `@{system_share_dirs}/glvnd/egl_vendor.d/`

### DISCUSSION THREAD

**Discussion by beroal** [roddhjav/apparmor.d#499] [Q&A] [answered]
Some programs (Sway, Alacritty, pavucontrol-qt) require

```
@{system_share_dirs}/glvnd/egl_vendor.d/{,*.json} r,
```

Which abstraction should I include? I already included `abstractions/desktop`.

**Comment by roddhjav** [comment, marked as answer]
This is vulkan data. It should already be included in the `vulkan-strict` abs. Usually you need to `graphics` abs: https://apparmor.pujol.io/development/abstractions/#graphics
